gitpod-io/gitpod
# [content-service-api] Fix codegen

## Description


Codegen wasn't correctly generating JS files, only TS definitions. This would result in the code compiling, but failing at runtime because the relevant methods actually didn't exist.

## Related Issue(s)

Fixes #

## How to test

1. Delete all generated files in `components/content-service-api/typescript/src` (excluding `index.ts`)
2. Run generation
3. Try to commit the diff, such that the pre-commit linter runs
4. Observe no diff

## Release Notes

```release-note
NONE
```

## Documentation


## Build Options:

      Experimental feature to run the build with GitHub Actions (and not in Werft).
      leeway-target=components:all
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

</details>

<details>
<summary>Installer Options</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
